apps/aehttp/src/sc_ws_api_jsonrpc.erl

```diff
@@ -234,6 +234,9 @@ process_incoming(Msg, FsmPid) ->
     end.
 
 -spec process_request(map(), pid()) -> no_reply | {reply, map()} | {error, term()}.
+process_request(#{<<"method">> := <<"channels.system">>,
+                  <<"params">> := #{<<"action">> := <<"ping">>}}, _FsmPid) ->
+    {reply, #{action => system, tag => pong}};
```

Comment:
What would be the behaviour from client side if client sent this before this code change?

I see that:
* `process_request /1` would return `{error, unhandled}`.
* `process_incoming/2` would therefore return `{reply, [{{error, unhandled}, #{<<"method">> := <<"channels.system">>, ...}}]}`. Though this callback is defined in behaviour `sc_ws_api` as [returning `{reply, map()} | no_reply | stop`](https://github.com/aeternity/aeternity/blob/de28b27c0fb627aadeaed42711bf4d8abcf7974a/apps/aehttp/src/sc_ws_api.erl#L40).

Could you please clarify behaviour on reception of this message before this change. And if possible also clarify return type of callback.

Reply:
Callback return value is ddressed.
If the ping message is received by a node without this change being applied, the node responds with
```
{  
   'channel_id':'ch_zVDx935M1AogqZrNmn8keST2jH8uvn5kmWwtDqefYXvgcCRAX',
   'error':{  
      'code':-32601,
      'message':'Method not found',
      'request':{  
         'jsonrpc':'2.0',
         'method':'channels.system',
         'params':{  
            'action':'ping'
         }
      }
   },
   'id':null,
   'jsonrpc':'2.0',
   'version':1
}```

Reply:
From https://www.jsonrpc.org/specification#error_object
> -32601 | Method not found ...

Ok.